website/src/js/components/SearchBar.js

```diff
@@ -60,8 +60,8 @@ class SearchBar extends Component {
         this.props.dispatch(setSearch('', starting_query));
       }
 
-      if (params.fields && params.fields.length > 0 ) {
-        this.props.dispatch(setActiveFields(params.fields));
+      if (params.active_field && params.active_field.length > 0 ) {
```

Comment:
Could the `active_field` parameter (both here and in the actual URL) be camelCase instead of snake_case. I know we're not 100% camelCase everywhere but I'm trying to fix that moving forward.